test/eventually.go

```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 )
 
-const iterations = 20
+const iterations = 100
```

Comment:
Let's split to 2 constants, one for `Eventually` and one for `Consistently` and not change the one for `Consistently`

The reason is that we pay the full time for `Consistently` every time so if you increase the iterations times 5, you make all tests that use `Consistently` 5 times longer.. this is painful